### PR TITLE
Google Analytics page tracking for DApp2

### PR DESCRIPTION
**Pull request by DanielVF**
Google Analytics page tracking for DApp2

Also, this ensure that we have a pre GraphQL query, top level page title for pages that will later change their title as the result of a GraphQL query (listings, offers). For discussion: sometimes GraphQL instantly returns cached responses, meaning that the final page title is sometimes the one recorded, rather than the pre-query one. Probably okay...

### CODE REVIEW

File: experimental/origin-dapp2/public/dev.html [hunk 1]
```diff
@@ -33,5 +33,17 @@
     localStorage.ognNetwork = 'localhost'
     localStorage.metaMaskEnabled = 'true'
   </script>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-106384880-2"></script>
```
Comment (nick) [inline experimental/origin-dapp2/public/dev.html]: I think we can remove from dev.html as that only gets used for local development

Comment (DanielVF) [reply experimental/origin-dapp2/public/dev.html]: Makes sense. We can still check that the analytics code is working by looking at `window.dataLayer` variable.

Comment (DanielVF) [reply experimental/origin-dapp2/public/dev.html]: I've removed this.

**Comment by DanielVF** [comment]
Anything else needed on this one, Nick?

**Comment by nick** [comment]
Nope I think it's good... looks like a few merge conflicts though. Will merge once those are resolved.

**Pull request merged**